src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -292,7 +295,7 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 	private scrollToActiveCell(): void {
 		// Get active cell from active notebook editor
 		const activeCellElement = document.querySelector(`.editor-group-container.active .notebook-cell.active`);
-		activeCellElement.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
+		activeCellElement.scrollIntoView({ behavior: 'auto', block: 'nearest' });
```

Comment:
What's this change for?

Reply:
The smooth effect takes longer get to the cell. That's why it looks like we are leaving behind the cell when we hold down the key.  Since we are selecting the next cell, we can scroll in one 'jump'. 

I'm going to remove the parameters though because that's the default behavior of scrollIntoView.